### PR TITLE
Update user-and-permission-management - TO REVIEW.md

### PR DESCRIPTION
**Pull request by molnarm84**

### CODE REVIEW

File: content/concepts/user-and-permission-management.md [hunk 1]
```diff
@@ -3,3 +3,14 @@ title: "User and Permission Management"
 metaTitle: "sensenet - User Management"
 metaDescription: "sensenet User Management"
 ---
+
+sensenet offers a sophisticated user and permission management system, giving you the possibility to apply permission rules according to the logic you prefer.
+
+# Why is it important?
+There are several situations when you have to adjust the rights of the users (when there are some changes in the company, e.g. someone leaves the company), and it is important to make it in a short time.
```
Comment (huszerlbence) [inline content/concepts/user-and-permission-management.md:10]:
```suggestion
There are several situations when you have to adjust user rights, someone leaves the company for example. In this case it is important to make it in a short time. 
```

File: content/concepts/user-and-permission-management.md [hunk 2]
```diff
@@ -3,3 +3,14 @@ title: "User and Permission Management"
...
 metaDescription: "sensenet User Management"
 ---
+
+sensenet offers a sophisticated user and permission management system, giving you the possibility to apply permission rules according to the logic you prefer.
+
+# Why is it important?
+There are several situations when you have to adjust the rights of the users (when there are some changes in the company, e.g. someone leaves the company), and it is important to make it in a short time. 
+
+# How does it work?
+In Sensenet it is possible to set permissions on role-based or on document level too. There are several built-in goups and users, and also permissions in sensenet, however it is possible to create custom ones as well.
```
Comment (huszerlbence) [inline content/concepts/user-and-permission-management.md:13]:
```suggestion
In sensenet it is possible to set permissions base on roles or on document level too. There are several built-in goups and users, and also permissions in sensenet, however it is possible to create custom ones as well.
```

File: content/concepts/user-and-permission-management.md [hunk 3]
```diff
@@ -3,3 +3,14 @@ title: "User and Permission Management"
...
+As sensenet is built around the concept “everything is content” user management is very simple, and thanks to the tree structure you can take advantage of the permission inheritance during your work.
```
Comment (huszerlbence) [inline content/concepts/user-and-permission-management.md:14]:
```suggestion
As sensenet is built around the concept “everything is content”, user management is very simple, and thanks to the tree structure you can take advantage of the permission inheritance during your work.
```

**Review by huszerlbence — CHANGES_REQUESTED** [review]
made some suggestions, check and approve if you agree

File: content/concepts/user-and-permission-management.md (hunk 2 above)
Comment (molnarm84) [reply content/concepts/user-and-permission-management.md:13]:
```suggestion
In sensenet it is possible to set permissions based on roles or on document level too. There are several built-in goups and users, and also permissions in sensenet, however it is possible to create custom ones as well.
```

**Review by herflis — APPROVED** [review]

**Review by huszerlbence — APPROVED** [review]

**Pull request merged**